locale/base-localization-file.yaml

```diff
@@ -0,0 +1,311 @@
+    languageSub: Sets the HTML Lang property for your brew. May affect hyphenation or spellcheck.
+    theme:
+    renderer:
+    Legacy:
+    v3:
+    legacyDemoLink: click here to see the demo page for the old legacy renderer!
+
+    authors: Authors
+    None.:
+    invited authors:
+    invite author:
+    invitedSub1: Invited authors are case sensitive.
+    invitedSub2: After adding an invited author, send them the edit link. there, they can choose to accept or decline the invitation.
+
+    privacy:
+    publish:
+    unpublish:
+    publishedSub: published homebrews will be publicly viewable and searchable (eventually...)
+    delete:
+    delete brew:
+    onlyAuthorDelete: Are you sure you want to delete this brew? Because you are the only owner of this brew, the document will be deleted permanently
+    confirm1: Are you REALLY sure? You will not be able to recover the document.
+    multipleAuthorDelete: Are you sure you want to remove this brew from your collection? This will remove you as an editor, but other owners will still be able to access the document.
+    confirm2: Are you REALLY sure? You will lose editor access to this document.
+    themesLegacy: Themes are not supported in the Legacy Renderer
+
+sharePage:
+  metadataWindow:
+    Authors:
+    no authors:
+    Systems:
+    no systems:
+    Tags:
+    no tags:
+    Description:
+    No description.:
+    Updated:
+
+  sourceDropdown:
+    source:
+    view:
+    download:
+    clone to new:
+
+accountPage:
+  Account Information:
+  username: "Username:"
+  noUser: No user currently logged in
+  lastLogin: "Last Login:"
+  Homebrewery Information:
+  brewsOnHomebrewery: "Brews on Homebrewery:"
+  Google Information:
+  linkedToGoogle: "Linked to Google:"
+  brewsOnGoogleDrive: "Brews on Google Drive:"
+  noFiles: Unable to retrieve files
+  followSteps: follow these steps to renew your Google credentials.
+  Default Save Location:
+  Homebrewery:
+  Google Drive:
+  Language Selection:
+  langSub: This option selects your language exclusively for the user Interface, and not for your brews. You can change brew language in their properties tab.
+
+errorCodes:
+  Crit Fail!:
+  error:
+  No error text:
+  unkownError: "## An unknown error occurred! \n\n We aren't sure what happened, but our server wasn't able to find what you were looking for."
+  googleError: "## An error occurred while retrieving this brew from Google Drive! \n\n Google reported an error while attempting to retrieve a brew from this link."
+  gd404: "## We can't find this brew in Google Drive! \n\nThis file was saved on Google Drive, but this link doesn't work anymore."
+  gd4042: "Note that this brew belongs to the Homebrewery account"
+  gd4043: which is
+  gd4044: your account.
+  gd4045: not your account (you are currently signed in as
+  gd4046: and you are not currently signed in to any account.
+  gd404longText: |
+    The Homebrewery cannot delete files from Google Drive on its own, so there
```

Comment:
I am unsure of THIS nomenclature

Reply:
I note that gd404* are simply successive strings which get stitched together with other values in the calling context. This suggests that maybe the `translate()` function should support passing in an object of keyed values which get substituted into the translated string.

Thus, the multiple `.translate()` calls here would instead become just one call, `"GDoc file not found error".translate({'account': props.brew.account});`, with all the magic happening in the function. 

This will also help when a translation needs to literally vary based on other variables e.g. plurality. Note that this will also easily handle cases where the variation is particular to the language and so the logic can't be done outside it (e.g. handling dual number plurality in Arabic).

Reply:
I now reckon we can push the PR like this and make that change later.